deleting-compaction-strategy/src/main/java/com/protectwise/cassandra/retrospect/deletion/CassandraPurgedData.java

```diff
@@ -0,0 +1,120 @@
+package com.protectwise.cassandra.retrospect.deletion;
+
+import com.protectwise.cassandra.util.SerializerMetaData;
+import org.apache.cassandra.db.Cell;
+import org.apache.cassandra.db.ColumnFamily;
+import org.apache.cassandra.utils.ByteBufferUtil;
+
+import java.io.Serializable;
+import java.nio.ByteBuffer;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Created by ayaz on 9/3/17.
+ */
+public class CassandraPurgedData implements Serializable {
+
+    private String ksName;
+
+    private String cfName;
+
+    private Set<String> partitonKeys = new HashSet<>();
+
+    private Set<String> clusterKeys = new HashSet<>();
+
+    private Map<String, SerializerMetaData> columnSerializerMetaDatas = new HashMap<>();
```

Comment:
cell is different from column, we have metaData information for each column, but not for cell. there could be more than one cell for a single column